tests/BUILD.bazel

```diff
@@ -1,22 +1,4 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
Commit message: Please consider mentioning that a new package was introduced.
Please also consider dropping the part about tests looking into implementation details - which albeit being true - does not help the reviewer understand why is this commit needed IMO.